src/app/productConfig/productConfigForm/productConfigForm.tpl.html

```diff
@@ -254,52 +257,50 @@ <h4 class="panel-title border-bottom-small mt" translate>
+                  <select class="form-control form-control-subtle"
+                          ng-model="$ctrl.itemConfig['RECURRING_START_MONTH']"
+                          ng-options="(m | date:'MM') as (m | date:'MMMM, yyyy') for m in $ctrl.possibleTransactionMonths($ctrl.nextDrawDate)"
+                          ng-change="$ctrl.changeStartDay($ctrl.itemConfig['RECURRING_DAY_OF_MONTH'], $ctrl.itemConfig['RECURRING_START_MONTH'])">
+                  </select>
+                </label>
               </div>
-              <div class="col-sm-2">
-                <div class="form-group">
-                  <label>
-                    <span translate>{{'DAY'}}</span>
-                    <select class="form-control form-control-subtle"
-                            ng-model="$ctrl.itemConfig['RECURRING_DAY_OF_MONTH']"
-                            ng-options="o as (o | ordinal) for o in $ctrl.possibleTransactionDays($ctrl.itemConfig['RECURRING_START_MONTH'], $ctrl.nextDrawDate)"
-                            ng-change="$ctrl.changeStartDay($ctrl.itemConfig['RECURRING_DAY_OF_MONTH'], $ctrl.itemConfig['RECURRING_START_MONTH'])">
-                    </select>
-                  </label>
-                </div>
+            </div>
+            <div class="col-sm-2">
+              <div class="form-group">
+                <label>
+                  <span translate>{{'DAY'}}</span>
+                  <select class="form-control form-control-subtle"
+                          ng-model="$ctrl.itemConfig['RECURRING_DAY_OF_MONTH']"
+                          ng-options="o as (o | ordinal) for o in $ctrl.possibleTransactionDays($ctrl.itemConfig['RECURRING_START_MONTH'], $ctrl.nextDrawDate)"
+                          ng-change="$ctrl.changeStartDay($ctrl.itemConfig['RECURRING_DAY_OF_MONTH'], $ctrl.itemConfig['RECURRING_START_MONTH'])">
+                  </select>
+                </label>
               </div>
-              <div class="col-sm-4">
+            </div>
+            <div class="col-sm-4">
+              <div class="form-group">
+                <label translate>{{'GIFT_START_DATE'}}</label>
                 <div class="form-group">
-                  <label translate>{{'GIFT_START_DATE'}}</label>
-                  <div class="form-group">
-                    {{$ctrl.startMonth( $ctrl.itemConfig['RECURRING_DAY_OF_MONTH'], $ctrl.itemConfig['RECURRING_START_MONTH'], $ctrl.nextDrawDate ).format('ll')}}
-                  </div>
+                  {{$ctrl.startMonth( $ctrl.itemConfig['RECURRING_DAY_OF_MONTH'], $ctrl.itemConfig['RECURRING_START_MONTH'], $ctrl.nextDrawDate ).format('ll')}}
                 </div>
               </div>
             </div>
-          </div><!-- // panel-body -->
-        </div><!-- // panel -->
+          </div>
+        </div><!-- // panel-body -->
       </div>
     </div>
-    <div class="panel panel-default give-modal-panel">
+    <div ng-if="$ctrl.useV3 !== 'true'" class="panel panel-default give-modal-panel">
```

Comment:
This is unrelated to your PR, so this change should be squashed with the first commit so it will just disappear.